services/src/main/java/org/keycloak/broker/saml/SAMLEndpoint.java

```diff
@@ -351,10 +351,10 @@ protected Response handleLoginResponse(String samlResponse, SAMLDocumentHolder h
                     return callback.error(relayState, Messages.IDENTITY_PROVIDER_UNEXPECTED_ERROR);
                 }
 
-                boolean assertionIsEncrypted = AssertionUtil.isAssertionEncrypted(responseType);
+                boolean assertionIsEncrypted = AssertionUtil.isAssertionsEncrypted(responseType);
 
                 if (config.isWantAssertionsEncrypted() && !assertionIsEncrypted) {
-                    logger.error("The assertion is not encrypted, which is required.");
+                    logger.error("The assertions are not encrypted, which is required.");
```

Comment:
Currently Keycloak only uses first assertion and this PR does not change this behaviour. Why to introduce this change?